docs/ABOUT.md

```diff
@@ -0,0 +1,19 @@
+[*Elixir*](http://elixir-lang.org/), initially released in 2012, extends upon the already robust features of Erlang while also being easier for beginners to access, read, test, and write. Everything in Elixir is an expression.
```

Comment:
_Everything in Elixir is an Expression._ seems like a non-sequitur right here. Let's stick that together with the stuff about regex and first-class documentation.